content/blog/pulkomandy/2021-06-03-haiku_activity_report_may_2021.md

```diff
@@ -8,157 +8,93 @@ tags = []
 
 Hello, it's time for the May activity report!
 
+Before starting the report, I would like to thank our donors for their donations. Your donations help us cover our expenses and help us reach our goal to hire people to work on Haiku full-time. I would also like to thank all the community for their countless hours of effort of implementing new features, triaging bugs, translating, supporting other users, and spreading the Haiku word all around.
```

Comment:
I am uncomfortable with the use of first person here, especially as this ends up on my blog and I didn't write this.